tests/dummy/app/routes/demos/sl-menu.js

```diff
@@ -5,7 +5,7 @@ export default Ember.Route.extend({
 
     model: function() {
         return  Ember.Object.create({ label: null, pages: [
-            Ember.Object.create({ label: 'Colors', classNames: [ 'current' ], pages: [
+            Ember.Object.create({ label: 'Colors', className: 'current', pages: [
```

Comment:
Prefer that these be specified as an array rather than a text string. It will be easier to dynamically manipulate an array of options passed to this component than a text string.